packages/aura-language-server/package.json

```diff
@@ -31,38 +31,39 @@
     "acorn-loose": "^6.0.0",
     "acorn-walk": "^6.0.0",
     "change-case": "^3.1.0",
-    "enhanced-resolve": "^2.2.2",
-    "fs-extra": "^5.0.0",
+    "enhanced-resolve": "^4.3.0",
+    "fs-extra": "^9.0.1",
```

Comment:
What about similar updates to `lightning-lsp-common` ? Some of the modules here are also used in that package.

Reply:
Luis, as we spoke yesterday, now all the versions are corresponding